test/example/c-api-multiple-tables.txt

```diff
@@ -0,0 +1,18 @@
+(module
+ (type $i32_i32_=>_i32 (func (param i32 i32) (result i32)))
+ (table $tab 1 1 funcref)
```

Comment:
Is this test supposed to have multiple tables?

Reply:
Updated!

Reply:
It would be good to add `adder` to the second table as well to prove that that works.

Reply:
Updated again.